custom_cards/custom_card_mpse_thermostat/custom_card_mpse_thermostat.yaml

```diff
@@ -23,27 +22,17 @@ custom_card_mpse_thermostat:
         type: "custom:button-card"
         template:
           - "icon_info"
-          - "ulm_language_variables"
+          - "ulm_translation_engine"
         tap_action:
           action: "more-info"
         entity: "[[[ return entity.entity_id ]]]"
         name: "[[[ return entity.name ]]]"
         label: >-
           [[[
-              var label = variables.ulm_off ;
-              if (entity.state =='off'){
-                label = variables.ulm_off ;
-              }else if (entity.state == 'heat'){
-                label = variables.ulm_heat ;
-              }else if (entity.state == 'cool'){
-                label = variables.ulm_cool ;
-              }else{
-                label = variables.ulm_idle;
-              }
               if(entity.attributes.temperature){
                 return (entity.attributes.current_temperature ) + '°' + ' • ' + label + ' (' + entity.attributes.hvac_action + ')';
```

Comment:
`label` is not specified anymore. Shouldn't it be `variables.ulm_translation_state`?

Reply:
Fixed, my bad!